apps/server/src/shared/domain/service/permission.service.ts

```diff
@@ -44,7 +46,13 @@ export class PermissionService {
 		return permissions;
 	}
 
-	hasUserAllSchoolPermissions(user: User, requiredPermissions: string[]): boolean {
+	/**
+	 * Determines whether a user has all the rights specified in requiredPermissions
+	 * @param user
+	 * @param requiredPermissions
+	 * @returns boolean
+	 */
+	hasAllPermissions(user: User, requiredPermissions: string[]): boolean {
```

Comment:
I dislike keeping the notion of global permissions. I would prefer these functions to be about "schoolpermissions", to facilitate a future switch to a scope based permission model

Reply:
We look into it in next iterations.